src/mobile/src/screens/AboutScreen.jsx

```diff
@@ -0,0 +1,126 @@
+          <Text>
+            In the early 2000s, before the rise of social media apps like Twitter and Facebook,
+            Planet solved an important problem in the free and open source community. It used
+            various "feed" technologies (RSS, Atom, CDF) to allow blog posts from different
+            platforms to be aggregated into a single page that was constantly updated with the
+            latest posts by people within a particular community. {'\n\n'}
+            Written in Python by Jeff Waugh and Scott James Remnant, Planet could be configured with
+            a list of blog feeds and an HTML template. It would use these to dynamically generate a
+            site with posts in chronological order from the specified feeds.
+          </Text>
+        </View>
+        <Text style={[styles.header, styles.paragraphHeader]}>In Search of a New Planet</Text>
+        <Text>
+          Our original Planet was shutdown in January 2020. The software we use was last updated 13
+          years ago. While the underlying code has drifted further into the past, our needs have
+          moved forward. Maintaining the existing system, especially with the number of students
+          involved in open source at Seneca, has become too difficult. Our current site often
+          breaks, and needs manual interventions on a regular basis. Going forward, we need a new
+          planet to call home.{'\n\n'}
+          We have decided it is time to consider moving to a new system. Unfortunately, almost every
+          system that came to replace Planet has itself become unmaintained.{'\n\n'}
+          Rather than try to find an existing solution, we have instead decided to try and create
+          one. Because we need this software, we also feel that we should create and maintain it.
+          And, since our need for a Planet comes out of our collective work on open source, we think
+          that creating it together as open source would be the most desireable path forward.
+        </Text>
+        <Text style={[styles.header, styles.paragraphHeader]}>Trying to Define Our Planet</Text>
+        <Text>
+          We have learned a number of things over the past decade running our own planet. We've also
+          watched as social media and modern technologies have reshaped our expectations for what a
+          system like this can and should be.{'\n\n'}
+          It's not 100% clear what we need to build, which is part of the fun. It is our hope that
+          you will leave a mark on this project, and bring your own ideas, experience, and code to
+          the task of defining our planet.
+        </Text>
+      </ScrollView>
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+  },
+  scrollView: {
+    width: '80%',
+  },
+  header: {
+    color: '#A0D1FB',
+    fontWeight: 'bold',
+    paddingVertical: 10,
+  },
+  aboutHeader: {
+    fontSize: 32,
```

Comment:
when it comes to font size, people usually use `em`? Should we do it here?

Reply:
Usually it will be `sp` or `dp` for mobile app but either of them or `em` work, I check the doc and it say `number`

Reply:
Sure!